tests/test_suites.py

```diff
@@ -4,15 +4,18 @@
 from dcqc.suites.suite_abc import SuiteABC
 from dcqc.suites.suites import FileSuite, OmeTiffSuite, TiffSuite
 from dcqc.tests.test_abc import TestABC, TestStatus
-from dcqc.tests.tests import FileExtensionTest, LibTiffInfoTest
+from dcqc.tests.tests import FileExtensionTest, GrepDateTest, LibTiffInfoTest
 
 FileType("None", ())
 FileType("Unpaired", ())
 
 
 class RedundantFileSuite(TiffSuite):
     file_type = FileType.get_file_type("None")
-    del_tests = (LibTiffInfoTest,)
+    del_tests = (
+        LibTiffInfoTest,
+        GrepDateTest,
```

Comment:
Note to self: It might be nice to be able to specify strings here so you don't have to import every test that you're adding or removing.

Reply:
Will create a ticket to track this.